core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -113,7 +113,7 @@ shared.belowInPercent=Below % from market price
 shared.aboveInPercent=Above % from market price
 shared.enterPercentageValue=Enter % value
 shared.OR=OR
-shared.notEnoughFunds=You don''t have enough funds in your Haveno wallet for this transaction—{0} is needed but only {1} is available.\n\nPlease add funds from an external wallet, or fund your Haveno wallet at Funds > Receive Funds.
+shared.notEnoughFunds=You don''t have enough funds in your Haveno wallet for this transaction?{0} is needed but only {1} is available.\n\nPlease add funds from an external wallet, or fund your Haveno wallet at Funds > Receive Funds.
```

Comment:
Looks like a find and replace error? These changes to `?` shouldn't be made.

Reply:
Entire displayStrings.properties file from latest was used.